### ISSUE TITLE
Some links are missing discernible text, but should have it for screen readers

### ISSUE THREAD

**Issue by bedmison** [OpenDSA/OpenDSA-LTI#191]
Some links are missing discernible text For example, the admin "wrench" or the exit/logout button. 

To solve this problem, you need to fix at least (1) of the following:

- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

And fix the following:

- Element is in tab order and does not have accessible text
